rest-servlet/src/main/java/org/hawkular/inventory/rest/RestResourceTypes.java

```diff
@@ -112,8 +113,9 @@ public ResourceType get(@PathParam("tenantId") String tenantId,
     public Set<Resource> getResources(@PathParam("tenantId") String tenantId,
                                       @PathParam("resourceTypeId") String resourceTypeId) {
 
-        return inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId)
-                .resources().getAll().entities();
+        ResourceTypes.Single single = inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId);
```

Comment:
This is a more general problem, IMHO, with the way we resolve `entities()` method in the tinkerpop impl. I am not sure we should resolve to one-off fixups like this, because the same issue is still present for all other entity types.